tests/unit/test_installation.py

```diff
@@ -327,3 +326,30 @@ def test_migrations_broken():
 
     with pytest.raises(IllegalState):
         installation.load(BrokenConfig)
+
+
+def test_enable_files_in_repos():
+    ws = create_autospec(WorkspaceClient)
+    ws.current_user.me().user_name = "foo"
+    installation = Installation(ws, "ucx")
+
+    # enableWorkspaceFilesystem is true
+    ws.workspace_conf.get_status.return_value = {"enableWorkspaceFilesystem": "true"}
+    installation._enable_files_in_repos()
```

Comment:
Don't test private methods directly - mock or assert api calls they make

Reply:
I have asserted the api call from inside _enable_files_in_repos(), but there is still a call to the private method.